github/tool.gpt

```diff
@@ -193,10 +209,24 @@ Tools: github.com/gptscript-ai/datasets/filter
 Name: List PRs for Review
 Description: Lists pull requests that are awaiting review by the authenticated user.
 Credential: ./credential
+Share Context: GitHub Context
 Tools: github.com/gptscript-ai/datasets/filter
 
 #!/usr/bin/env node ${GPTSCRIPT_TOOL_DIR}/index.js listPRsForReview
 
+---
+Name: GitHub Context
+Type: context
+
+#!sys.echo
+
+## Instructions for using GitHub tools
```

Comment:
I think we use `START INSTRUCTIONS: <tool name(s)>` elsewhere

Reply:
looks like weve done that in just a couple places - jira, database, knowledge, and workspace files.

the format i used here is more common, but there is probably some value in standardizing on something.